docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,772 @@
+* [NEW (new-style XML)](audit-log-filter-new.md) and 
+* [JSON](audit-log-filter-json.md).
+
+### `audit_log_filter_format_unix_timestamp`
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-format-unix-timestamp |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | OFF  |
+
+This option is only supported for JSON-format files.
+
+Enabling this option adds a `time` field to JSON-format files. The integer represents the UNIX timestamp value and indicates the date and time when the audit event was generated. Changing the value causes a file rotation because all records must either have or do not have the `time` field. This option requires the `AUDIT_ADMIN` and `SYSTEM_VARIABLES_ADMIN` privileges.
+
+This option does nothing when used with other format types.
+
+### `audit_log_filter_handler`
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-handler |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | String  |
+| Default | FILE  |
+
+Defines where the plugin writes the audit log filter file. The following values are available:
+
+* `FILE` - plugin writes the log to a location specified in [`audit_log_filter_file`](#audit_log_filter_file)
+* `SYSLOG` - plugin writes to the syslog
+
+
+### `audit_log_filter_key_derivation_iterations_count_mean`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-key-derivation-iterations-count-mean |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 60000  |
+| Minimum value | 1000 |
+| Maximum value | 1000000 |
+
+Defines the mean value of iterations used by the password-based derivation routine while calculating the encryption key and iv values. A random number represents the actual iteration count and deviates no more than 10% from this value.
+
+### `audit_log_filter_max_size`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-max-size |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 0 |
+| Minimum value | 0 |
+| Maximum value | 18446744073709551615 |
+| Unit | bytes |
+| Block size | 4096 |
+
+Defines pruning based on the combined size of the files:
+
+* 0 (zero), the default value, disables pruning based on size.
+* A value greater than 0 (zero) enables pruning based on size and defines the combined size limit. When the files exceed this limit, they can be pruned.
+
+The value is based on 4096 (block size). A value is truncated to the nearest multiple of the block size. If the value is less than 4096, the value is treated as 0 (zero).
+
+If the values for `audit_log_filter_rotate_on_size` and `audit_log_filter_max_size` are greater than 0, we recommend that `audit_log_filter_max_size` value should be at least seven times the `audit_log_filter_rotate_on_size` value. A lower value generates a warning to the error log and may remove all the rotated logs each time.
```

Comment:
Please do not mention warning to the error log here, we don't have such validation for those vars.